src/controllers/petsController.js

```diff
@@ -1,47 +1,55 @@
 const petsModel = require('../models/petsModel.js');
 const { inputValidation } = require("../utils/tools.js");
 const { ContentTypeError, PropNullorEmptyError, PropRequiredError } = require("../utils/errors.js");
+const { Logger } = require("../utils/log4js.js");
+const log = Logger();
 
 const {
-  readAllPets,
-  createNewPet
+  getAllPets,
+  createNewPet,
+  getPetById,
+  deletePetById,
+  updatePetById
 } = petsModel;
 
 const requiredFields = [
-  'TypeOfAnimal',
-  'Breed',
-  'Sex',
-  'Size',
-  'ShelterID',
-  'Picture',
-  'Availability',
-  'GoodWithOtherAnimals',
-  'GoodWithChildren',
-  'MustBeLeashed',
-  'Neutered',
-  'Vaccinated',
-  'HouseTrained',
-  'Description'
+  'test',
```

Comment:
What does 'test' do in this scenario?

Reply:
Good catch lol I forgot to remove it after testing